src/ImageSharp/Formats/Jpeg/Components/Decoder/ProfileResolver.cs

```diff
@@ -24,6 +24,14 @@ internal static class ProfileResolver
         (byte)'J', (byte)'F', (byte)'X', (byte)'X', (byte)'\0'
     };
 
+    /// <summary>
+    /// Gets the \n[ID or 10 91 73 68 32 specific markers.
+    /// </summary>
+    public static ReadOnlySpan<byte> OSQidMaker => new[]
+    {
+        (byte)'\n', (byte)'[', (byte)'I', (byte)'D', (byte)' '
+    };
```

Comment:
```suggestion
    public static ReadOnlySpan<byte> OSQidMaker => new[] { (byte)'\n', (byte)'[', (byte)'I', (byte)'D', (byte)' ' };
```
I'd like to have it written as one-liner, because when we are able to use ut8-literals it's easier to find & replace these.

Reply:
It's a good suggestion, but the other properties of the class already follow this same pattern, so I chose to keep it.